tests/test_description.py

```diff
@@ -1,6 +1,8 @@
-import pytest
+from functools import partial
 
-from busypie import ConditionTimeoutError, ONE_HUNDRED_MILLISECONDS, wait_at_most
+import pytest
+from busypie import (ONE_HUNDRED_MILLISECONDS, ConditionTimeoutError,
```

Comment:
why parenthesis ?

Reply:
This was applied by the "sort imports" feature of VS Code.

Reply:
Interesting